scipy/integrate/_quadpack_py.py

```diff
@@ -298,7 +298,9 @@ def quad(func, a, b, args=(), full_output=0, epsabs=1.49e-8, epsrel=1.49e-8,
     >>> integrate.quad(invexp, 0, np.inf)
     (1.0, 5.842605999138044e-11)
 
-    >>> f = lambda x,a : a*x
+    Calculate :math:`\\int^1_0 a x dx` for :math:`a = 1, 3`
```

Comment:
Adding a small space helps the readability. Here and below
```suggestion
    Calculate :math:`\\int^1_0 a x \, dx` for :math:`a = 1, 3`
```